create_student.rb

```diff
@@ -7,6 +8,11 @@ def initialize(person, app_instance)
     @app_instance = app_instance
   end
 
+  # method to save student to 'books.json'
+  def save_persons_to_json(persons)
+    File.write('person.json', JSON.pretty_generate(persons.map(&:to_hash)))
+  end
```

Comment:
- [ ] Hey Team! 👋 Stellar job on implementing the method for saving person to JSON. 📚 However, there's a slight deviation from the project requirement. 🚧 According to the specifications, we're supposed to save to JSON only on app exit, not immediately upon person creation. 🔄 Kindly review the current implementation and make the necessary adjustments to align with the project's directive. Let's ensure we save that data only when the app bids us farewell. 🌟 Thanks for your attention to detail! 🙌